### PR TITLE
Ret 5133

### PR DESCRIPTION
**Pull request by lvo700**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RET-5133

### Change description ###

Create Task "MultiplesReviewReferralJudiciary" for Judge (MUL WA)

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by JackReevies — APPROVED** [review]

File: src/main/resources/wa-task-completion-employment-et_englandwales_multiple.dmn [hunk 1]
```diff
@@ -21,6 +21,19 @@
           <text>"Auto"</text>
         </outputEntry>
       </rule>
+      <rule id="DecisionRule_00owiss">
+        <inputEntry id="UnaryTests_0l84new">
+          <text>"preAcceptanceCase",
```
Comment (fergusoshea) [inline src/main/resources/wa-task-completion-employment-et_englandwales_multiple.dmn:26]: AFAIK, preAcceptanceCase is not an event on Multiples. So no need to have it here. It was probably a copy and paste from the singles.

File: src/test/java/uk/gov/hmcts/et/taskconfiguration/dmn/EmploymentTaskCompletionMultipleTestEW.java [hunk 2]
```diff
@@ -28,13 +28,26 @@ public static void initialization() {
 
     static Stream<Arguments> scenarioProvider() {
         return Stream.of(
+            Arguments.of(
+                "preAcceptanceCase",
```
Comment (fergusoshea) [inline src/test/java/uk/gov/hmcts/et/taskconfiguration/dmn/EmploymentTaskCompletionMultipleTestEW.java:32]: As above

File: src/test/java/uk/gov/hmcts/et/taskconfiguration/dmn/EmploymentTaskCompletionMultipleTestScot.java [hunk 3]
```diff
@@ -28,13 +28,26 @@ public static void initialization() {
 
     static Stream<Arguments> scenarioProvider() {
         return Stream.of(
+            Arguments.of(
+                "preAcceptanceCase",
```
Comment (fergusoshea) [inline src/test/java/uk/gov/hmcts/et/taskconfiguration/dmn/EmploymentTaskCompletionMultipleTestScot.java:32]: As above

File: src/test/java/uk/gov/hmcts/et/taskconfiguration/dmn/EmploymentTaskConfigurationMultipleTestScot.java [hunk 4]
```diff
@@ -186,15 +186,14 @@ void when_taskId_then_return_workType(String taskType, List<Map<String, String>>
     }
 
     public static Stream<Arguments> workType_ScenarioProvider() {
-        List<Map<String, Object>> routineWork = List.of(Map.of(
+        List<Map<String, Object>> decisionMakingWork = List.of(Map.of(
```
Comment (fergusoshea) [inline src/test/java/uk/gov/hmcts/et/taskconfiguration/dmn/EmploymentTaskConfigurationMultipleTestScot.java:189]: Should this not be in addition to, not replacing, the existing tests?

File: src/test/java/uk/gov/hmcts/et/taskconfiguration/dmn/EmploymentTaskConfigurationMultipleTestScot.java [hunk 5]
```diff
@@ -220,21 +219,21 @@ void when_taskId_then_return_roleCategory(String taskType, List<Map<String, Stri
...
     public static Stream<Arguments> roleCategory_ScenarioProvider() {
-        List<Map<String, Object>> administrator = List.of(Map.of(
+        List<Map<String, Object>> judicial = List.of(Map.of(
             "name", "roleCategory",
-            "value", "ADMIN",
+            "value", "JUDICIAL",
             "canReconfigure", true
         ));
-        List<Map<String, Object>> legalOps = List.of(Map.of(
+        List<Map<String, Object>> administrator = List.of(Map.of(
```
Comment (fergusoshea) [inline src/test/java/uk/gov/hmcts/et/taskconfiguration/dmn/EmploymentTaskConfigurationMultipleTestScot.java:227]: Do we need to keep LegalOps here?

File: src/test/java/uk/gov/hmcts/et/taskconfiguration/dmn/EmploymentTaskConfigurationMultipleTestScot.java [hunk 6]
```diff
@@ -220,21 +219,21 @@ void when_taskId_then_return_roleCategory(String taskType, List<Map<String, Stri
...
+            "value", "ADMIN",
             "canReconfigure", true
         ));
 
         return Stream.of(
-            Arguments.of("ReviewReferralAdminMultiple", administrator),
+            Arguments.of("ReviewReferralJudiciaryMultiple", judicial),
+            Arguments.of("ReviewReferralAdminMultiple", administrator)
 
-            Arguments.of("ReviewReferralLegalOpsMultiple", legalOps)
```
Comment (fergusoshea) [inline src/test/java/uk/gov/hmcts/et/taskconfiguration/dmn/EmploymentTaskConfigurationMultipleTestScot.java:237]: Do we need to keep LegalOps here?

File: src/test/java/uk/gov/hmcts/et/taskconfiguration/dmn/EmploymentTaskConfigurationMultipleTestScot.java [hunk 7]
```diff
@@ -276,7 +275,7 @@ public static Stream<Arguments> description_ScenarioProvider() {
 
         return Stream.of(
             Arguments.of("ReviewReferralAdminMultiple", descReferralTab),
-            Arguments.of("ReviewReferralLegalOpsMultiple", descReferralTab)
+            Arguments.of("ReviewReferralJudiciaryMultiple", descReferralTab)
```
Comment (fergusoshea) [inline src/test/java/uk/gov/hmcts/et/taskconfiguration/dmn/EmploymentTaskConfigurationMultipleTestScot.java:278]: Do we need to keep LegalOps here?

File: src/test/java/uk/gov/hmcts/et/taskconfiguration/dmn/EmploymentTaskConfigurationMultipleTestScot.java [hunk 8]
```diff
@@ -431,11 +430,11 @@ public static Stream<Arguments> priority_ScenarioProvider() {
                          dueDateIntervalDays2NoReconfigure, defaultMajorPriorityNoReconfigure,
                          defaultMinorPriorityNoReconfigure, null, priorityDateOriginEar
             ),
-            Arguments.of("ReviewReferralLegalOpsMultiple", IS_URGENT,
+            Arguments.of("ReviewReferralJudiciaryMultiple", IS_URGENT,
```
Comment (fergusoshea) [inline src/test/java/uk/gov/hmcts/et/taskconfiguration/dmn/EmploymentTaskConfigurationMultipleTestScot.java:433]: Do we need to keep LegalOps here?

File: src/test/java/uk/gov/hmcts/et/taskconfiguration/dmn/EmploymentTaskConfigurationMultipleTestScot.java [hunk 9]
```diff
@@ -431,11 +430,11 @@ public static Stream<Arguments> priority_ScenarioProvider() {
...
                          dueDateIntervalDays1NoReconfigure, urgentMajorPriority, urgentMinorPriority,
                          null, priorityDateOriginEar
             ),
-            Arguments.of("ReviewReferralLegalOpsMultiple", NOT_URGENT,
+            Arguments.of("ReviewReferralJudiciaryMultiple", NOT_URGENT,
```
Comment (fergusoshea) [inline src/test/java/uk/gov/hmcts/et/taskconfiguration/dmn/EmploymentTaskConfigurationMultipleTestScot.java:437]: Do we need to keep LegalOps here?

File: src/test/java/uk/gov/hmcts/et/taskconfiguration/dmn/EmploymentTaskInitiationMultipleTestEW.java [hunk 10]
```diff
@@ -24,9 +24,8 @@ class EmploymentTaskInitiationMultipleTestEW extends DmnDecisionTableBaseUnitTes
 
     public static final String REFERRAL_ADMIN =
         HelperService.createReferrals("Referral Subject 1", "Referral Subject 2", "Admin", "Yes", "", "");
-
-    public static final String REFERRAL_LEGALOPS =
-        HelperService.createReferrals("Referral Subject 1", "Referral Subject 2", "Legal officer", "Yes", "", "");
+    public static final String REFERRAL_JUDGE =
```
Comment (fergusoshea) [inline src/test/java/uk/gov/hmcts/et/taskconfiguration/dmn/EmploymentTaskInitiationMultipleTestEW.java:27]: Do we need to keep LegalOps here?

File: src/test/java/uk/gov/hmcts/et/taskconfiguration/dmn/EmploymentTaskInitiationMultipleTestEW.java [hunk 11]
```diff
@@ -50,16 +49,15 @@ public static Stream<Arguments> scenarioProvider() {
             Arguments.of(
                 "createReferral",
                 null,
-                HelperService.mapAdditionalData(REFERRAL_LEGALOPS),
+                HelperService.mapAdditionalData(REFERRAL_JUDGE),
```
Comment (fergusoshea) [inline src/test/java/uk/gov/hmcts/et/taskconfiguration/dmn/EmploymentTaskInitiationMultipleTestEW.java:52]: Do we need to keep LegalOps here?

File: src/test/java/uk/gov/hmcts/et/taskconfiguration/dmn/EmploymentTaskConfigurationMultipleTestScot.java (hunk 5 above)
Comment (lvo700) [reply src/test/java/uk/gov/hmcts/et/taskconfiguration/dmn/EmploymentTaskConfigurationMultipleTestScot.java]: Fixed

**Pull request merged**